### ISSUE TITLE
FO4/FO4 Edit won't load through MO2.0.4 and things that do load won't recognize any mods

### ISSUE THREAD

**Issue by rfitzpat** [TanninOne/modorganizer#592]
Things like LOOT and Bodyslide don't recognize any of the MO loaded virtual files.  When I try to run FO4 through MO, either via the game exe or F4SE_Loader it gets to the end of the "Please Standy" screen then CTD every time.  I have no issue running F4SE directly.  Seems like the problem is something to do with my MO Install.

The only block in the MO Logs that seems to indicate the issue is:

> 11:29:26.293 <9456:10088> [D] NtCreateFile [source=\??\C:\Program Files (x86)\Steam\steamapps\common\Fallout 4] [rerouted=\??\C:\Program Files (x86)\Steam\steamapps\common\Fallout 4] [CreateDisposition=2] [*FileHandle=0000009EC0C3D8E0] [res=exists already]
11:29:26.293 [W] failed to create intermediate directory "Users": 123
11:29:26.293 <9456:10088> [D] CreateDirectoryW [reroute.fileName()=C:\Users\XXXXXXXX] [res=0]
11:29:26.298 <9456:10088> [D] NtCreateFile [source=\??\C:\Program Files (x86)\Steam\steamapps\common\Fallout 4] [rerouted=\??\C:\Program Files (x86)\Steam\steamapps\common\Fallout 4] [CreateDisposition=2] [*FileHandle=0000009EC0C3EBF8] [res=exists already]
11:29:26.298 <9456:10088> [D] CreateDirectoryW [reroute.fileName()=C:\] [res=1]
11:29:26.299 <9456:10088> [D] NtCreateFile [source=\??\C:\Program Files (x86)\Steam\steamapps\common\Fallout 4] [rerouted=\??\C:\Program Files (x86)\Steam\steamapps\common\Fallout 4] [CreateDisposition=2] [*FileHandle=0000009EC0C3EBF8] [res=exists already]
11:29:26.299 [W] failed to create intermediate directory "Users": 123
11:29:26.299 <9456:10088> [D] CreateDirectoryW [reroute.fileName()=C:\Users] [res=0]
11:29:26.299 <9456:10088> [D] NtCreateFile [source=\??\C:\Program Files (x86)\Steam\steamapps\common\Fallout 4] [rerouted=\??\C:\Program Files (x86)\Steam\steamapps\common\Fallout 4] [CreateDisposition=2] [*FileHandle=0000009EC0C3E9F0] [res=exists already]
11:29:26.299 [W] failed to create intermediate directory "Users": 123
11:29:26.299 <9456:10088> [D] CreateDirectoryW [reroute.fileName()=C:\Users\XXXXXXXX] [res=0]

Now it seems like if it's failing to create the virtual directory then that could explain why it doesn't seem to let LOOT or BodySlider see any of the files.  It could also point to why it's crashing.  Unfortunately I'm not sure why it's having the issue.  It's running as admin and I've tried installing as a dedicated version from the game folder as well as more generically from just C:\Mod Organizer

Same issue either way.  Thoughts?

**Comment by Sir1Twist** [comment]
Here is how to get BodySlide to work with MO2a4.
1) Outside of MO2A4, run Bodyslide x64. Then close it.
2) Open MO2a4, go to your drop down list on the right hand side, next to run. Edit the Bodyslide, and remove.
3) Go to the Data tab underneath that, go down to Tools, then Bodyslide, then you should see Bodyslide x64.exe. Right click only on that one. Then add as Executable.
4) Try running the newly made Bodyslide. If fail, don't worry, this can happen.
5) Run the Bodyslide x64 outside of MO2A4, again, this time, go to Settings.
6) Make sure the target game is Fallout 4, and set the Game Data Path to this:
D:\Fallout 4\steamapps\common\Fallout 4\Data\ (You will have a different path, but this will give you an idea where to go.) (Also, I don't install any games under the Program Files (x86) as that give some heaaches down the road.)
7) Hit Ok, then close Bodyslide. Then rerun the one in MO2A4. And you should be good to go.
8) If you still are getting problems, then add this under the drop down menu's edit, and Bodyslide, Start in line:
 (Again your path will be different, but this will help point the way.)
D:\Fallout 4\steamapps\common\Fallout 4\Mod Organizer 2 alpha 4\mods\BodySlide and Outfit Studio\Tools\BodySlide
9) Go to where you have the mod installed, and make sure you have the Body Slide x64 running as administrator, by right-clicking on the file, then Properties, Compatibility tab, and checking Run as Administrator.
If all goes well, you should be able to use Bodyslide in MO2a4. I will warn you sometimes it won't start, or stay running. But if you get it to run once, it will do so again. Also, before you modify anything in Bodyslide, switch back to Mod Organizer, and Unlock it. In case Bodyslide does lock up, which can happen, it won't crash Mod Organizer, as well.

**Comment by rfitzpat** [comment]
Actually BodySlider works to the point where I can tweak vanilla assets.  It just doesn't seem to see anything added by mods activated in MO.  Even in the set up directions you have to point the EXE loader to the place where the mod is actually at rather than the "data\whatever\" folder.

It just seems like nothing is seeing the virtual stuff.

'

**Comment by Sir1Twist** [comment]
I think I know what's going on here. Check MO to see if your clothing mods are active, or not. That will be in the left hand pane. And on the right hand pane, if they have a .esp. I know a few mods do. Making sure those are active, try opening Body slide again.

**Comment by rfitzpat** [comment]
the bodyslide thing is just a tiny thing and sort of works.  I'm much more concerned that even with no mods loaded loading Fallout 4 through MO makes my game CTD almost instantly.

**Comment by Sir1Twist** [comment]
I've not had any problems with my game running MO2.

**Comment by rfitzpat** [comment]
Yeah, I'm sure it's something I've got set wrong but I can't figure out what.

2 questions that might help me.  

Where should the QT archive files be going?
When I execute the usvfs_proxy exe it also just immediately says throws up the stopped responding box and does nothing.  Is that normal?

**Comment by bobmajdakjr** [comment]
i know there is a feature branch titled new vfs so he might be addressing a lot of problems already, because all the sudden it seems like a *lot* of people are having issues almost like it was a date based bomb.

**Comment by rfitzpat** [comment]
That makes some sense.  I hear a lot of people having issues like mine that aren't posting.  Seems like there is at least a small contingent that use this version of MO just as a literal mod organizer.  Track their mods, updates and so forth. But all as "unmanaged" mods.  I eagerly await another release.
